wic/imx-imx-boot-bootpart.wks.in

```diff
@@ -10,11 +10,11 @@
 #  - ---------- -------------- --------------
 # ^ ^          ^              ^              ^
 # | |          |              |              |
-# 0 |        8MiB          72MiB          72MiB + rootfs + IMAGE_EXTRA_SPACE (default 10MiB)
+# 0 |        8MiB          264MiB         264MiB + rootfs + IMAGE_EXTRA_SPACE (default 10MiB)
```

Comment:
Hi Otavio, we were surprised that the partition is now 333, not 256. The part command includes a default factor of 1.3, which explains the difference. https://docs.yoctoproject.org/dev/ref-manual/kickstart.html#command-part-or-partition.